api/node/src/routes.ts

```diff
@@ -3,12 +3,14 @@ import { CategoryScale, Colors } from 'chart.js';
 import Chart from 'chart.js/auto';
 import { getString, convertChartJsConfigToBase64String, generateBase64, sendImage } from './utils';
 import { ChartTypes } from './charts';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
 
 Chart.register(CategoryScale);
 Chart.defaults.animation = false;
 Chart.defaults.responsive = false;
 Chart.defaults.maintainAspectRatio = true;
 Chart.register(Colors);
+Chart.register(ChartDataLabels);
```

Comment:
Gives us some extra features for labeling, however it defaults to adding labels, which means most charts have to specify to return `null`.  It's annoying, but not certain how we can make it better.  Also need to figure out a reliable way to serialize functions so that they can be passed in as configuration.